docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -276,7 +275,7 @@ Mapping character types
 ^^^^^^^^^^^^^^^^^^^^^^^
 
 Trino's ``VARCHAR(n)`` maps to ``VARCHAR2(n CHAR)`` if ``n`` is no greater than
-4000. A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
+1.    A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
```

Comment:
Revert. This was correct. Oracle has two VARCHAR types and we use decide which to use depending on length of the VARCHAR on our side - 4000 characters being the magic value.

Reply:
Just a line below this is also incorrect -

```
Trino's ``CHAR(n)`` maps to ``CHAR(n CHAR)`` if ``n`` is no greater than 2000.
```

should be changed to

```
Trino's ``CHAR(n)`` maps to ``CHAR(n CHAR)`` if ``n`` is no greater than 500.
```

Reply:
thanks @hashhar, good catch. I forgot that whenever you open the Oracle doc it makes that change, its some weird bug.

@mosabua, hashhar found a second error in the Oracle doc. Should I make the update with this PR or a separate one?

Reply:
Personally I would take it out from this PR and then send a second PR for the value changes. Whatever we do however .. lets get this PR merged asap @electrum